Dockerfile

```diff
@@ -1,12 +1,3 @@
-FROM alpine:3.10.1
+FROM alpine:3.12.0
 ARG DUB_OPTIONS
-RUN apk --no-cache add build-base git libsodium-dev openssl openssl-dev sqlite-dev zlib-dev
-COPY --from=bpfk/pkgbuilder:latest --chown=root:root /home/effortman/.abuild/*.rsa.pub /etc/apk/keys/
-COPY --from=bpfk/ldc:latest /root/packages/ /root/packages/
-COPY --from=bpfk/dub:latest /root/packages/ /root/packages/
-RUN apk --no-cache add /root/packages/effortman/x86_64/ldc-1.16.0-r0.apk \
-    /root/packages/effortman/x86_64/ldc-runtime-1.16.0-r0.apk \
-    /root/packages/effortman/x86_64/ldc-static-1.16.0-r0.apk \
-    /root/packages/effortman/x86_64/dub-1.17.0-r0.apk \
-    /root/packages/effortman/x86_64/dtools-rdmd-2.087.1-r0.apk \
-    && rm -rf /root/packages/
+RUN apk --no-cache add build-base dtools-rdmd dub git ldc libsodium-dev openssl openssl-dev sqlite-dev zlib-dev
```

Comment:
Newline. Also you can remove `DUB_OPTIONS`, it's only useful in Agora. Last but not least do we need `openssl` ? I think I tested recently and removing it didn't have any effect. If memory serves me well, it used to be mandatory because of a bug in the version detection in Vibe.d but now works.